tests/modules/ecs6-class/line.test.js

```diff
@@ -2,13 +2,20 @@ const  Line = require('../../../modules/ecs6-class/line');
 const Point = require('../../../modules/ecs6-class/point');
 let line;
 describe('constructor', () => {
+    it('should be ok when all paramter is ok', () => {
+        line = new Line({ point1: new Point(1, 2), point2: new Point(4, 5), n: 4, slope: 7 });
+        expect(line.point1).toEqual(new Point(1, 2));
+        expect(line.point2).toEqual(new Point(4, 5))
+        expect(line.n).toBe(4);
+        expect(line.slope).toBe(7);
```

Comment:
more examples for good cases

Reply:
I would appreciate a more detailed explanation please.

Reply:
The teacher Can respond to this PR please?